lib/fastly/lib/index.js

```diff
@@ -9,16 +9,17 @@
  * Dependencies
  */
 const request = require('request');
-const Q = require('q');
 const debug = require('debug')('fastly');
 
 /**
  * Constructor
  */
-function Fastly (apikey, service, options) {
+function Fastly (apikeys, service, options) {
 
 	var opts = options || {};
-	this.apikey = apikey || '';
+	this.apikey = apikeys.shift();
+	this.apikeys = apikeys || [];
```

Comment:
apikeys should be defaulted to an array before any array commands are called on it. 
```
this.apikeys = apikeys || [];
this.apikey = apikeys.shift();
```